src/main/java/com/nvs/store/controllers/ProductController.java

```diff
@@ -29,18 +31,20 @@ public ResponseEntity<Product> getProduct(@PathVariable Long id) {
         return ResponseEntity.ok(productService.getProduct(id).get());
     }
 
+
     @PostMapping
-    public ResponseEntity<Product> addProduct(@RequestBody Product product) {
-        productService.addProduct(product);
-        return ResponseEntity.status(CREATED).body(productService.getProduct(product.getId()).get());
+    @ResponseStatus(CREATED)
```

Comment:
scope of your ticket to write unit tests - not change existent implementation. please remove changes out of test package

Reply:
We changed it with mentors a little , because it was hard to test and tests failed